pkg/instance/log.go

```diff
@@ -20,7 +21,17 @@ func (s *Server) LogSetLevel(ctx context.Context, req *rpc.LogSetLevelRequest) (
 }
 
 func (ops V1DataEngineInstanceOps) LogSetLevel(ctx context.Context, req *rpc.LogSetLevelRequest) (resp *emptypb.Empty, err error) {
-	/* TODO: Implement this */
+	// Set instance-manager log level.  We expect a string such as "debug", "info", or "warn".
+	oldLevel := logrus.GetLevel()
+	newLevel, err := logrus.ParseLevel(req.Level)
+	if err != nil {
+		return nil, err
+	}
+	if oldLevel != newLevel {
+		logrus.Warnf("Updating log level from %v to %v", oldLevel, newLevel)
+		logrus.SetLevel(newLevel)
+	}
+
```

Comment:
v2's LogSetLevel only handles spdk_tgt's log level, so the PR needs to update the instance-manager pod's log level as well.

Reply:
I see.